rs/nns/governance/canister/canister.rs

```diff
@@ -161,15 +164,22 @@ fn set_governance(gov: Governance) {
 fn schedule_timers() {
     schedule_seeding(Duration::from_nanos(0));
     schedule_adjust_neurons_storage(Duration::from_nanos(0), NeuronIdProto { id: 0 });
+    schedule_prune_following(Duration::from_secs(0));
 }
 
 // Seeding interval seeks to find a balance between the need for rng secrecy, and
 // avoiding the overhead of frequent reseeding.
 const SEEDING_INTERVAL: Duration = Duration::from_secs(3600);
 const RETRY_SEEDING_INTERVAL: Duration = Duration::from_secs(30);
+const PRUNE_FOLLOWING_INTERVAL: Duration = Duration::from_secs(1);
 
-fn schedule_seeding(duration: Duration) {
-    ic_cdk_timers::set_timer(duration, || {
+// Once this amount of instructions is used by the
+// Governance::prune_some_following, it stops, saves where it is, schedules more
+// pruning later, and returns.
+const MAX_PRUNE_SOME_FOLLOWING_INSTRUCTIONS: u64 = 25_000_000;
```

Comment:
How many neurons would be processed at this limit?  We may want to increase this.

Reply:
I think we can conservatively estimate that a neuron takes 2e6 instructions to pull from stable memory. So, in total, it would take 400e9 instructions to pull 200 kiloneurons. If each batch uses at least 25e6 instructions, then it will take 16 kilobatches. If one batch takes one second, then it would take < 4.5 hours to complete a full pass. Seems reasonable.

So, that's 5.4 full passes per day. 5.4 * 400e9 = 2.16e12. Assuming 1 instruction is APPROXIMATELY 1 cycle, that's a couple of bucks per day. Seems reasonable. Let me add these back of the envelop calculations to a comment...